pkg/k8shandler/cluster.go

```diff
@@ -31,11 +30,11 @@ func nodeMapKey(clusterName, namespace string) string {
 }
 
 // CreateOrUpdateElasticsearchCluster creates an Elasticsearch deployment
-func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateElasticsearchCluster() error {
-	esClient := elasticsearchRequest.esClient
+func (er *ElasticsearchRequest) CreateOrUpdateElasticsearchCluster() error {
```

Comment:
:tada: thank you!!!!

Reply:
because it comes from receiving an `elasticsearch` cr event